README.md

```diff
@@ -1,10 +1,4 @@
-<div style="position: relative; width: 100%; height: 0; padding-top: 25.0000%;
- padding-bottom: 0; box-shadow: 0 2px 8px 0 rgba(63,69,81,0.16); margin-top: 1.6em; margin-bottom: 0.9em; overflow: hidden;
- border-radius: 8px; will-change: transform;">
-  <iframe loading="lazy" style="position: absolute; width: 100%; height: 100%; top: 0; left: 0; border: none; padding: 0;margin: 0;"
-    src="https:&#x2F;&#x2F;www.canva.com&#x2F;design&#x2F;DAFOJ6HT5so&#x2F;watch?embed" allowfullscreen="allowfullscreen" allow="fullscreen">
-  </iframe>
-</div>
+![PROJECT-0-matic](https://user-images.githubusercontent.com/80773310/194979405-9df5df2d-d368-41bf-8ca8-e4aa040a6fc1.gif)
```

Comment:
I tried updating the readme gif from my main. Hopefully it will work at your side too

Reply:
I'm curious, what happens if you just add this line

![](https://user-images.githubusercontent.com/80773310/194979405-9df5df2d-d368-41bf-8ca8-e4aa040a6fc1.gif)

Reply:
Whoops, you won't see the text but basically removed PROJECT-0-MATIC from the start of the gif link and use only []

then it should work

Reply:
Hmm, interesting; I wonder how you turn gifs into links such as GitHub user content because, on my readme, I only use drag and drop for the gif. That's why I am having a hard time when trying to implement this into other person repos.